### PR TITLE
LRQA-53958 | Master

### PR DESCRIPTION
**Pull request by nkkwon**

### CODE REVIEW

**Comment by alexcandidos** [comment]
Thanks for the new script @nkkwon !!

**Comment by alexcandidos** [comment]
ci:test:app-builder

**Comment by alexcandidos** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
ci:test:app-builder

**Comment by alexcandidos** [comment]
ci:test:app-builder

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 1]
```diff
@@ -868,6 +868,23 @@ definition {
 		AppBuilderAdmin.deleteEntry(itemName = "my custom object");
 	}
 
+	@description = "Delete an existing Form View with no active publishing through the kebab/action menu"
+	@priority = "5"
+	test DeleteFormViewThroughActionMenu {
+		property portal.acceptance = "true";
```
Comment (alexcandidos) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Hey @nkkwon , could you please change it to "false"? Thanks!

Comment (nkkwon) [reply modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Pushed newest changes

**Comment by alexcandidos** [comment]
Thanks @nkkwon !!

**Comment by alexcandidos** [comment]
Pull request forwarded to https://github.com/brianchandotcom/liferay-portal/pull/82289. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**